docs/iris/src/userguide/loading_iris_cubes.rst

```diff
@@ -236,7 +236,7 @@ explicitly enabled by setting the "cell_datetime_objects" option in :class:`iris
 
     >>> filename = iris.sample_data_path('uk_hires.pp')
     >>> cube_all = iris.load_cube(filename, 'air_potential_temperature')
-    >>> print 'All times :\n', cube_all.coord('time')
+    >>> print('All times :\n' + str(cube_all.coord('time')))
```

Comment:
It's a shame we have to add the `str`. We could use `format` instead but I'm not sure it's worth the trouble.